src/host/main.cpp

```diff
@@ -422,6 +422,18 @@ int main(int argc, char** argv)
     ->capture_default_str()
     ->check(CLI::NonexistentPath);
 
+  CurveID curve_id = CurveID::SECP384R1;
+  std::vector<std::pair<std::string, CurveID>> curve_id_map = {
+    {"secp384r1", CurveID::SECP384R1}, {"secp256r1", CurveID::SECP256R1}};
+  app
+    .add_option(
+      "--curve-id",
+      curve_id,
+      "Elliptic curve to use for node's certificate (used for TLS and ledger "
```

Comment:
Node and network identity in the case of a node starting in new or recovery mode.